docs/usage/notes.rst

```diff
@@ -278,10 +281,19 @@ won't free disk space, they merely tag data as deleted in a new transaction.
 Be aware that as soon as you write to the repo in non-append-only mode (e.g. prune,
 delete or create archives from an admin machine), it will remove the deleted objects
 permanently (including the ones that were already marked as deleted, but not removed,
-in append-only mode).
+in append-only mode). Automated edits to the repository (such as a cron job running
+``borg prune``) will render append-only mode moot if data is deleted.
 
-Note that you can go back-and-forth between normal and append-only operation by editing
-the configuration file, it's not a "one way trip".
+Even if an archive appears to be available, it is possible an attacker could delete
+just a few chunks from an archive and silently corrupt its data. While in append-only
+mode, this is reversible, but ``borg check`` should be run before a writing/pruning
+operation on an append-only repository to catch accidental or malicious corruption::
+
+    # run on the repository host, without append-only mode
+    borg check --verify-data repo && borg compact repo
```

Comment:
repo host = machine that runs borg serve?

one often does not have the key there, so you can't run borg check for archives there.

Reply:
I forgot the terminology aside from "host," I'll change that.

I didn't consider that `borg check` would need to decrypt—I forgot the "A" of "AEAD" requires decryption. I guess I'll just say "run without append-only mode".

Reply:
it really needs to decrypt, so it is also the "E".

so just say "run without append-only mode" or "... in non-append-only mode".